### ISSUE TITLE
[TSDoc> Intro]: While navigating the 'Intro' page, screen reader is conveying the same name as 'Copy code to clipboard button'.

### ISSUE THREAD

**Issue by vagpt** [microsoft/rushstack-websites#246]
**Test Environment:**
Version: Version 24H2 (OS Build 26231.5000)
Browser Version: 127.0.2638.2 (Official build) dev (64-bit)
URL: https://tsdoc.org/
Screen Reader: Narrator

**Repro Steps:**
1. Open the above URL in edge dev browser and login with v-.
2. Turn on the narrator through Ctrl+ windows+ Enter key.
3. Navigate to the 'Intro' link and press enter key to activate it.
4. Turn on the scan mode.
5. Press 'B' key and observe the issue.

**Actual Result:**
While navigating the 'Intro' page, the screen reader is conveying the same name as 'Copy code to clipboard button'.
Also, while navigating using 'B' key, screen reader is also conveying the same name as 'Copy code to clipboard button' due to this user is not able to differntiate between both buttons.

**Expected Result:**
Different or Unique name should be defined for the both 'Copy to clipboard' button.

**Note:**
Same issue is observed with NVDA screen reader as well.

**User Impact:**
It will impact the screen reader user as they will not able to differentiate between both buttons if the same will be defined on the page.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section. 

![Same name is defined for copy code to clipboard button](https://github.com/microsoft/rushstack-websites/assets/95913748/72d9474e-6a5f-467c-aec6-b8cd80bd788f)

https://github.com/microsoft/rushstack-websites/assets/95913748/86de317e-14e0-44e6-af90-9a04831d4344
